extensions/resource-deployment/src/ui/model.ts

```diff
@@ -32,6 +33,24 @@ export class Model {
 		return value === undefined ? defaultValue : value === 'true';
 	}
 
+	public getCodeCellContentForNotebook(): string[] {
+		const statements: string[] = Object.keys(this.propValueObject)
+			.filter(propertyName => propertyName.startsWith(NoteBookEnvironmentVariablePrefix))
+			.map(propertyName => {
+				const value = this.escapeForNotebookCodeCell(this.getStringValue(propertyName, ''));
+				const regex = new RegExp(`^${NoteBookEnvironmentVariablePrefix}`);
+				const varName = propertyName.replace(regex, '').toLocaleLowerCase();
+				return `${varName} = '${value}'`;
+			});
+		statements.push(`print('Variables have been set successfully.')`);
+		return statements.map(line => line + EOL);
```

Comment:
Does the EOL not get inserted automatically when you pass in an array of strings? Seems like that would be the behavior I expect.

If you do need to add them here just append the EOL when you create the strings above (lines 43 and 45)

Reply:
fixed